jaxrs-spec/src/main/asciidoc/chapters/providers/_exceptionmapper.adoc

```diff
@@ -30,6 +30,13 @@ superclass of the exception. If two or more exception providers are
 applicable, the one with the highest priority MUST be chosen as
 described in <<provider_priorities>>.
 
+A JAX-RS implementation MUST include a default exception mapping provider
+that implements ExceptionMapper<Throwable> and which SHOULD
```

Comment:
Just a minor formatting note:

```suggestion
that implements `ExceptionMapper<Throwable>` and which SHOULD
```

Maybe we should use backticks here like we do it for all other classes?